src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -122,97 +123,114 @@ const WorkflowDashboard = _.flow(
 
   return div({ style: { padding: '1rem 2rem 2rem', flex: 1, display: 'flex', flexDirection: 'column' } }, [
     workflowDetailsBreadcrumbSubtitle(namespace, name, submissionId, workflowId),
-    workflow === undefined ?
-      h(Fragment, [
+    Utils.cond([
+      workflow === undefined,
+      () => h(Fragment, [
         div({ style: { fontStyle: 'italic', marginBottom: '1rem' } }, [`Fetching workflow metadata...`]),
         centeredSpinner()
-      ]) :
-      h(Fragment, [
+      ])
+    ], [
+      metadataArchiveStatus === 'ArchivedAndDeleted',
+      () => h(Fragment, [
         div({ style: { fontStyle: 'italic', marginBottom: '1rem' } }, [`Workflow metadata fetched in ${fetchTime}ms`]),
```

Comment:
Should this be removed? I feel it's confusing to see `.. metadata fetched in x seconds` if the next line says 'Workflow Details Archived'. I do agree that technically UI did get some metadata back, but this might be confusing for users no?